### DISCUSSION TITLE
Start_geopix.bat

### DISCUSSION THREAD

**Discussion by sjenkin** [EnviralDesign/GeoPix#12] [Q&A]
Hello,

I have touchdesigner 2021 installed, and when I ran the bat file to start geopix, it searches for a version form 2020. I editted to bat file to search for the latest version of touchdesigner.

Is there any issues with newer version of touchdesigner or should I be all good?

Cheers

**Comment by EnviralDesign** [comment]
Hi @sjenkin sorry for the delay here just now seeing this. As a matter of fact I just got GeoPix upgraded to work with the latest 2021 version.

There were lots of small and medium problems to sort out which is why I was putting it off. Should be fully compatible now :)

**Comment by sjenkin** [comment]
Awesome, will test it out asap
